src/screens/CreatePost/elements/UserProfile.js

```diff
@@ -103,13 +103,14 @@ const UserProfile = ({
       <View style={styles.container}>
         {userProfile()}
         <ToggleSwitch
-          isOn={isAnonymous}
-          onColor={colors.blue}
-          label="Anonymity"
-          offColor="#F5F5F5"
-          size="small"
-          labelStyle={styles.switch}
-          onToggle={() => setTypeUser(!isAnonymous)}
+          value={isAnonymous}
+          onValueChange={() => setTypeUser(!isAnonymous)}
+          labelLeft="Anonymity"
+          backgroundActive={colors.lightgrey}
+          backgroundInactive={colors.lightgrey}
+          circleInActiveColor={colors.blue1}
+          inactiveTextColor={colors.blue1}
+          styleLabelLeft={styles.switch}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The properties passed to the `ToggleSwitch` component have been significantly altered. Make sure these changes do not affect the functionality or appearance of the toggle switch in undesired ways. Also, ensure that the new properties (`labelLeft`, `backgroundActive`, `backgroundInactive`, `circleInActiveColor`, `inactiveTextColor`) are properly handled in the `ToggleSwitch` component.
```diff
-         <ToggleSwitch
-          isOn={isAnonymous}
-          onColor={colors.blue}
-          label="Anonymity"
-          offColor="#F5F5F5"
-          size="small"
-          labelStyle={styles.switch}
-          onToggle={() => setTypeUser(!isAnonymous)}
-         />
+         <ToggleSwitch
+           value={isAnonymous}
+           onValueChange={() => setTypeUser(!isAnonymous)}
+           labelLeft="Anonymity"
+           backgroundActive={colors.lightgrey}
+           backgroundInactive={colors.lightgrey}
+           circleInActiveColor={colors.blue1}
+           inactiveTextColor={colors.blue1}
+           styleLabelLeft={styles.switch}
+         />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->